jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -358,6 +359,17 @@ def parameterized(harness_group: Iterable[Harness],
   for is_stable in [False, True]
 )
 
+lax_linalg_qr = tuple(
+  Harness(f"multi_array_shape={jtu.format_shape_dtype_string(shape, dtype)}_fullmatrices={full_matrices}",
+          lax_linalg.qr,
+          [RandArg(shape, dtype), StaticArg(full_matrices)],
+          shape=shape,
+          dtype=dtype,
+          full_matrices=full_matrices)
+  for dtype in [jnp.float32, jnp.float64, jnp.complex64, jnp.complex128] # See jax.lib.lapack.geqrf
```

Comment:
I think it would be better to run the test for all types, or at least all floating point ones, and then check that either Jax throws an error or else jax2tf produces the right result.

Reply:
Addressed in latest commit.